ui/ajax/submit-editor.js

```diff
@@ -11,22 +12,38 @@ const { CoarseChannel } = require(`../../lib/model.js`);
  * @param {object} request Passed from Express.
  * @param {object} response Passed from Express.
  */
-module.exports = function addFixtures(request, response) {
+module.exports = async function addFixtures(request, response) {
```

Comment:
Since this is a temporary solution, I wouldn't split it up into different API endpoints. The API documentation is definitely something that has to be done eventually, but it shouldn't be part of this PR.